src/highdicom/content.py

```diff
@@ -711,14 +712,72 @@ def __init__(
 
         """  # noqa: E501
         super().__init__()
+        self._issuer_of_identifier = issuer_of_identifier
         if issuer_of_identifier_type is None:
             self.LocalNamespaceEntityID = issuer_of_identifier
+            self._issuer_of_identifier_type = None
         else:
             self.UniversalEntityID = issuer_of_identifier
             issuer_of_identifier_type = UniversalEntityIDTypeValues(
                 issuer_of_identifier_type
             )
             self.UniversalEntityIDType = issuer_of_identifier_type.value
+            self._issuer_of_identifier_type = issuer_of_identifier_type
+
+    @property
+    def issuer_of_identifier(self) -> str:
+        """str: Identifier of the issuer."""
+        return self._issuer_of_identifier
+
+    @property
+    def issuer_of_identifier_type(self) -> Union[
+        UniversalEntityIDTypeValues, None
+    ]:
+        """highdicom.UniversalEntityIDTypeValues: Type of the issuer."""
+        return self._issuer_of_identifier_type
+
+    @classmethod
+    def from_dataset(
+        cls,
+        dataset: Dataset,
+    ) -> 'IssuerOfIdentifier':
+        """Construct object from an existing dataset.
+
+        Parameters
+        ----------
+        dataset: pydicom.dataset.Dataset
+            Dataset
+
+        Returns
+        -------
+        highdicom.IssuerOfIdentifier
+            Issuer of identifier
+
+        """
+        issuer_of_identifier = deepcopy(dataset)
```

Comment:
For performance reasons, we have been adding the ability to bypass the copy into all `from_dataset` and `from_sequence` methods. I think weh should do the same here.

```suggestion
    @classmethod
    def from_dataset(
        cls,
        dataset: Dataset,
        copy: bool = True,
    ) -> 'IssuerOfIdentifier':
        """Construct object from an existing dataset.

        Parameters
        ----------
        dataset: pydicom.dataset.Dataset
            Dataset
        copy: bool
            If True, the underlying dataset is deep-copied such that the
            original dataset remains intact. If False, this operation will
            alter the original dataset in place.

        Returns
        -------
        highdicom.IssuerOfIdentifier
            Issuer of identifier

        """
        if copy:
            issuer_of_identifier = deepcopy(dataset)
        else:
            issuer_of_identifier = dataset
```


 (p.s. feel free to just accept the suggestion in github's interface).

Reply:
pps this won't be used furing the call from SpecimenDescription.from_dataset() yet, but I can add that later

Reply:
Accepted and tested the suggestion.